### DISCUSSION TITLE
I have a doubt?

### DISCUSSION THREAD

**Discussion by Jyoti-prakash-rout** [North-East-India-Community/welcome#3] [Q&A] [answered]
Does this program eligible for students from Odisha?

**Comment by TheSuranaverse** [comment]
No, it's only for students or working professionals from N.E. states.
Hope this answered your question 😊 
If yes, then please mark it as answered!
Thank you for reaching out!!

**Comment by siddharthdayalwal** [comment, marked as answer]
> Does this program eligible for students from Odisha?

It is currently only for folks residing in any of the North-East Indian states: Arunachal Pradesh, Assam, Manipur, Meghalaya, Mizoram, Nagaland, Sikkim, Tripura, West Bengal
